PSCompatibilityCollector/Microsoft.PowerShell.CrossCompatibility/Commands/ConvertJsonCommands.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Management.Automation;
+using Microsoft.PowerShell.CrossCompatibility.Data;
+using Microsoft.PowerShell.CrossCompatibility.Retrieval;
+using Microsoft.PowerShell.CrossCompatibility.Utility;
+using Newtonsoft.Json;
+
+namespace Microsoft.PowerShell.CrossCompatibility.Commands
+{
+    /// <summary>
+    /// Class defining the ConvertTo-PSCompatibilityJson cmdlet.
+    /// Turns given .NET objects into JSON strings.
+    /// </summary>
+    [Cmdlet(VerbsData.ConvertTo, CommandUtilities.MODULE_PREFIX + "Json")]
+    [OutputType(typeof(string))]
+    public class ConvertToPSCompatibilityJsonCommand : PSCmdlet
+    {
+        private JsonProfileSerializer _serializer;
+
+        /// <summary>
+        /// The object to convert to JSON.
+        /// </summary>
+        [Parameter(Position = 0, Mandatory = true, ValueFromPipeline = true)]
+        public object[] Item { get; set; }
+
+        /// <summary>
+        /// If set, do not include whitespace like indentation or newlines in the output JSON.
+        /// </summary>
+        [Parameter]
+        [Alias("Compress")]
```

Comment:
I actually put this in because [ConvertTo-Json calls this `Compress`](https://github.com/PowerShell/PowerShell/blob/687697bc90879fe0162d08d409485632d9e79e79/src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs#L45-L52), but left it an alias because I think it's a bad name for it

Reply:
Happy to go either way on this btw